src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/DebugTokenReplacer.cs

```diff
@@ -66,15 +66,15 @@ internal class DebugTokenReplacer : IDebugTokenReplacer
             // Since Env variables are an immutable dictionary they are a little messy to update.
             if (resolvedProfile.EnvironmentVariables != null)
             {
-                foreach (var kvp in resolvedProfile.EnvironmentVariables)
+                foreach (System.Collections.Generic.KeyValuePair<string, string> kvp in resolvedProfile.EnvironmentVariables)
                 {
                     resolvedProfile.EnvironmentVariables = resolvedProfile.EnvironmentVariables.SetItem(kvp.Key, await ReplaceTokensInStringAsync(kvp.Value, true).ConfigureAwait(false));
                 }
             }
 
             if (resolvedProfile.OtherSettings != null)
             {
-                foreach (var kvp in resolvedProfile.OtherSettings)
+                foreach (System.Collections.Generic.KeyValuePair<string, object> kvp in resolvedProfile.OtherSettings)
```

Comment:
add using

Reply:
Okay, I get it, will run simplification fixer over it. It didn't catch this...

Reply:
Oh this is https://github.com/dotnet/roslyn/issues/24034, ugh, this is all manual.